src/components/CustomStatusBar/index.android.js

```diff
@@ -1,18 +0,0 @@
-import React from 'react';
-import {StatusBar} from 'react-native';
-import themeColors from '../../styles/themes/default';
-
-/**
- * Only the Android platform supports "setBackgroundColor"
- */
-
-export default class CustomStatusBar extends React.Component {
```

Comment:
Agree with deleting this, not sure why we used it in the first place. Maybe a future improvement is to remove it entirely and use native iOS styling instead.